Sources/Scout/Implementations/XML/PathExplorerXML+Get.swift

```diff
@@ -66,15 +66,15 @@ extension PathExplorerXML {
 
     // MARK: - Dictionary
 
-    /// - parameter ignoreArraySlicing: If `true`, the fact that self is part of a slicing will be ignore to simple get the index
-    func get(for key: String) throws  -> PathExplorerXML {
+    /// - parameter detailedName: If`true`, when using a dictionary filter, the keys names will be changed to reflect the filtering
```

Comment:
Missing a space after `If`.

Reply:
Fixed in latest commit 739ca313022c667ae53fc96f79a81c50d6a8264f